### ISSUE TITLE
Forever saying "Please answer (y)es or (n)o."

### ISSUE THREAD

**Issue by Botspot** [jmcerrejon/PiKISS#188]
```
$ curl -sSL https://git.io/JfAPE | bash
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
Please answer (y)es or (n)o.
```
Run in a terminal on a system without dialog installed. How long has this been broken?

**Comment by Botspot** [comment]
This bug also occurs when trying to install PiKISS from Pi-Apps.

**Comment by Botspot** [comment]
More specifically, this error only happens when there is already a `~/piKiss` folder in existence, the `dialog` package is uninstalled, and you try to run the installation command. `curl -sSL https://git.io/JfAPE | bash`
I encountered this error on a new system where I had the piKiss folder in my home directory from an older install, then tried to install PiKISS. This would also happen if a user first clones your repo with git and then runs your installation script.

**Comment by jmcerrejon** [comment]
Don't know how long this it happens, because I don't use Pi-Apps. I'll check it out, anyway.

**Comment by Botspot** [comment]
It can be caused without pi-apps. It happens when you run the install command when a pikiss folder already exists in $HOME.

**Comment by jmcerrejon** [comment]
Fixed 9b4d87a75ce5f3131304896076a7cc65e4557338

Thanks for the report! ☺️
